### PR TITLE
chore(deps): use full semver docker tag for node images

### PR DESCRIPTION
**Pull request by jrasm91**
Migrate to tags with full semver (with the idea that renovate PRs should update the tags with the actual version bumps, instead of just changing the hash).

If this works, I can make a separate PR in the base-images repo.

### CODE REVIEW

**Review by danieldietzler — APPROVED** [review]
Why haven't we had this already? lmao

**Review by bo0tzz — APPROVED** [review]
Definitely better to set the full tag, yeah.

Note that pinDigests is enabled so renovate will create PRs to add shas again:
https://github.com/immich-app/immich/blob/d7a33c8ec24bb66bd82472bf58045442349bca05/renovate.json#L3

**Comment by danieldietzler** [comment]
> Definitely better to set the full tag, yeah.
> 
> Note that pinDigests is enabled so renovate will create PRs to add shas again:
> 
> https://github.com/immich-app/immich/blob/d7a33c8ec24bb66bd82472bf58045442349bca05/renovate.json#L3

That probably was intentionally left out as it's just easier to let renovate add the hashs later than having to look them up yourself :D

**Comment by bo0tzz** [comment]
I thought the intention was to remove the hash, that's why I mentioned it. If we wanna keep it then that's fine of course.

File: server/Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ COPY --from=dev /usr/src/app/node_modules/@img ./node_modules/@img
 COPY --from=dev /usr/src/app/node_modules/exiftool-vendored.pl ./node_modules/exiftool-vendored.pl
 
 # web build
-FROM node:iron-alpine3.18@sha256:53108f67824964a573ea435fed258f6cee4d88343e9859a99d356883e71b490c as web
+FROM node:20.13.1-alpine3.18
```
Comment (bo0tzz) [inline server/Dockerfile:28]:
```suggestion
FROM node:20.13.1-alpine3.18 as web
```

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Oopsie

**Review by zackpollard — APPROVED** [review]

**Review by bo0tzz — APPROVED** [review]

**Pull request merged**